libclc/clc/include/clc/math/clc_subnormal_config.h

```diff
@@ -1,24 +1,10 @@
-/*
- * Copyright (c) 2015 Advanced Micro Devices, Inc.
- *
- * Permission is hereby granted, free of charge, to any person obtaining a copy
- * of this software and associated documentation files (the "Software"), to deal
- * in the Software without restriction, including without limitation the rights
- * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
- * copies of the Software, and to permit persons to whom the Software is
- * furnished to do so, subject to the following conditions:
- *
- * The above copyright notice and this permission notice shall be included in
- * all copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
- * THE SOFTWARE.
- */
+//===- clc_subnormal_config.h ---------------------------------------------===//
```

Comment:
FWIW, we nowadays no longer require the file name to be part of the header comment (see https://llvm.org/docs/CodingStandards.html#file-headers for the current standard header).

Reply:
Oh thanks that's good to know. I might just remove it then as I don't think it helps much.

Is the second doxygen section containing `\file` etc. required here? I assume not since you didn't point it out (and components like MLIR don't seem to include it), but the documentation you linked doesn't make it sound optional. I'm not sure libclc is even built with doxygen.

Reply:
It's not required, and I don't think you can really add a meaningful file-level comment in this kind of mass-change anyway.

Reply:
Yeah it would have made this patch a lot harder! That's done now.